docs/nodle-sdk/nodle-sdk-unity.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 7
+---
+
+# Nodle SDK - Unity
+
+Integrating the Nodle SDK into your Unity app is easy and straightforward. You can use our [NodleSDK Quickstart application](https://github.com/NodleCode/nodlesdk-unity/tree/master/Unity_NodleSDK_v0.6) or just follow the steps below and you should be ready to go in no time.
+
+## Step 1: Generate Developer Key
+- Go and [Create a Web Wallet](nodle-wallets/polkadot-js/how-to-create-a-nodle-cash-wallet.md)
+- Make sure you save your private key!
+- Copy your ```public_key``` from the extension
+- Proceed forward with next steps.
+
+## Step 2: Setup Developer environments
+Please make sure to follow our [Android SDK](nodle-sdk-android-integration.md) and [iOS SDK](nodle-sdk-ios-integration.md) and configure your environement based on the requirements there.
+
+## Step 3: Create Unity application
+You can follow the steps to create a new project or skip below to just importing the plugin depending on your platform.
+
+### Creating a new project - Android
+Please start by creating a new project Android it is not required to use that template you can create a blank or 3D or other.
+
+![Empty Import](/img/docs/nodle-sdk/unity-android-step1.png)
```

Comment:
You might want to change `Empty Import` to something slightly more legible

Reply:
done